Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -641,6 +641,22 @@ public extension Purchases {
         return try await logOutAsync()
     }
 
+    ///
+    /// Updates the current appUserID to a new one, without associating the two.
+    /// - Important: This method is **only available** in Custom Entitlements Computation mode.
+    /// Receipts posted by the SDK to the RevenueCat backend after calling this method will be sent
+    /// with the newAppUserID.
+    ///
+    @objc(switchUserTo:)
```

Comment:
I think in `Obj-C` this would be better as `switchUserToNewAppUserID:` (the conventions are different in both languages).